pcsx2/gui/MainFrame.cpp

```diff
@@ -411,7 +411,7 @@ MainEmuFrame::MainEmuFrame(wxWindow* parent, const wxString& title)
 	m_menuSys.Append(MenuId_Boot_CDVD2,		_("Initializing..."));
 
 	m_menuSys.Append(MenuId_Boot_ELF,		_("&Run ELF..."),
-		_("For running raw PS2 binaries directly"));
+		_("For running raw PS2 binaries directly."));
```

Comment:
Ah I didn't know that, I thought the translations looked up the lines only. Well I prefer to have periods because it looks a little bit more proper. I'll wait to see what the others have to say about it.